types/ibc.go

```diff
@@ -118,11 +118,13 @@ type IBCReceiveResult struct {
 type IBCReceiveResponse struct {
```

Comment:
Could you update the link to the Rust `IbcReceiveResponse` to the latest version in the doc comment?